ospo-book/content/en/06-chapter.md

```diff
@@ -1,7 +1,103 @@
+
+### The Commons Dilemma
+`🏛 Social-Gov-Oriented`
+
+The tragedy of the commons is notably present in the OSS world, where disproportionate consumption by certain users exacerbates sustainability issues. Discussions suggested the need to strengthen infrastructure support to mitigate these disparities and ensure a more balanced contribution and consumption (also called usage) model within the OSS community.
+
+### Community Norms and Funding Strategies
+`🏛 Social-Gov-Oriented`
+
+Establishing community norms around the costs associated with SBOM delivery and compliance has been proposed as an idea. This could involve suggesting a donation model based on organization size to account for the rising costs of compliance. Additionally, the role of risk mitigation as a motivation for organizations to invest in OSS was discussed, alongside the potential for government agencies to contribute indirectly to OSS projects through third parties.
+
+### Policy and Funding Initiatives
+`🏛 Social-Gov-Oriented`
+
+The possibility of the EU mandating open source as a requirement for Horizon funding suggests a growing recognition of the value of OSS at the policy level. There is an ongoing discussion on strategic funding models, like the German sovereign tech fund, and the potential for government roles in supporting OSS.
+
+### Challenges in OSS Sustainability
+`📈 Business-Oriented`
+
+The underfunding of crucial projects like Django, and the broader issue of maintainers being underpaid or volunteering, underscores the sustainability challenges within OSS. The critical role of vulnerability management and the need for corporate contributions to OSS have been highlighted as areas requiring attention and action.
+
+### Training and Education as Funding Avenues
+`🏛 Social-Gov-Oriented`
+
+The potential for integrating OSS training within government contracts is seen as a means to provide both funding for OSS organizations and valuable training for employees. This represents an approach to leveraging existing procurement processes to support OSS sustainability.
+
+
+## OSPOs for Container Capabilities
+
+Content has been curated from the [OSPOlogyLive Apeldoorn Roundtable Discussion](https://github.com/todogroup/ospology/discussions/438#discussioncomment-8620412) and based on the adoption of open source software (OSS) for container orchestration, particularly Kubernetes, and its implications for IT strategy in organizations.
+
+### OSPOs working with Container Hosting Infrastructure Teams
+
+Teams dedicated to this infrastructure, focusing on aspects such as image repositories and platform enablement, collaborate closely with OSPOs to support container infrastructure and capabilities. Initiatives like the Cloud Platform Engineering Team (CPET) has been highlighted for their efforts in publishing self-built operators, such as the Platform as a Service (PaaS) Operator, which automates tasks like initiating namespaces or projects on Kubernetes platforms.
```

Comment:
What are the Cloud Platform Engineering Team and the PaaS Operator?

Reply:
Agreed. A bit more context would help.

Reply:
this was taken from the notes from the "Building an Open Source Foundation for Your Container Capabilities" roundtable. probably a Dutch initiative, but I've asked the people who participated to unblock this and hopefully get the right links where these initiatives are.

Reply:
[They replied](https://thetodogroup.slack.com/archives/C042VQXFZU6/p1711635547832109?thread_ts=1711385027.470979&cid=C042VQXFZU6)! 

According to the article, CPET is a team within the Dutch tax admin.

To learn more about the team's responsibilities, see reference [here](https://striive.com/nl/opdrachten/capgemini-nederland-bv---yf/lead-transformatie/capgemini-nederland-bv---yf/senior-openshift-engineer---belastingdienst/31e003f6-1a1a-4321-961b-5a1c217c8930)

> The Tax and Customs Administration has an IV-Container Platform Enable Team (CPET) team at the Generieke Facilities unit. This team is responsible for:

>Making available an operational Container Platform for Cloud-Native applications that has different (production) environments.
> Onboarding DevOps teams on the platform through intakes and guidance.
> Making the (conforming policy) available of current generic images and operators for new-build Cloud Native applications and (medium of Lift&Shift) migration of Java applications.
> Providing consultancy and sharing of best and good practices with regard to the realization, supply and operation of Cloud-Native applications.
> Investigating and testing new capabilities.




operator Project as a Service (PaaS)

see reference [here](https://tweakers.net/plan/4002/de-belastingdienst-als-it-werkgever-cloud-native-wordt-de-standaard.html):

> Although we brought all our products with the GitOps principles, there was a separate application process for each product. It was not easy for the teams and ourselves. For that reason, we have built our own operator, so that we can offer our products in an unambiguous way and further automate production within our cloud environment. Teams can now configure the desired capabilities with a custom resource configuration. The PaaS operator ensures that they are automatically available on the platform. In fact, this marked the transition to Cloud 2.0, making far easier enablement on a large scale. We have called the operator Project as a Service (laughing: not to be confused with Platform as a Service) and we hope to reap the benefits of this in the near future